disnake/colour.py

```diff
@@ -278,12 +278,15 @@ def greyple(cls) -> Self:
 
     @classmethod
     def dark_theme(cls) -> Self:
-        """A factory method that returns a :class:`Colour` with a value of ``0x36393F``.
+        """A factory method that returns a :class:`Colour` with a value of ``0x313338``.
         This will appear transparent on Discord's dark theme.
 
         .. versionadded:: 1.5
+
+        .. versionchanged:: 2.10
+            Updated from ``0x36393F`` to ``0x313338`` to reflect theme changes.
```

Comment:
I'm not sure whether we should add these notes, they can make the docstring really long in the future

Reply:
that's fair -- updated the docstring.